examples/transformerconfigs/README.md

```diff
@@ -84,12 +95,14 @@ nameReference:
   - path: spec/containers/env/valueFrom/secretKeyRef/name
     version: v1
     kind: Pod
-    (etc.)    
+  # add additional paths to resource fields
 ```
 
-## customizing transformer configurations
+## Customizing transformer configurations
+
```

Comment:
Suggest to change it to following paragraphs.

Kustomize has a default set of transformer configurations. You can view the default transformer configurations by saving them
to a local directory by calling `kustomize config save -d`. 

Kustomize also supports adding extra transformer configurations by adding configuration files in the kustomization.yaml file. This tutorial shows how to customize those configurations to: